### PR TITLE
opencl: update 2022.01.04

### PR DESCRIPTION
**Pull request by 3rav**
Update PKGBUILD

Update PKGBUILD

### CODE REVIEW

**Comment by Biswa96** [comment]
Separate commits with proper commit message would be better.

File: mingw-w64-opencl-headers/PKGBUILD [hunk 1]
```diff
@@ -3,7 +3,7 @@
 _realname=opencl-headers
 pkgbase=mingw-w64-${_realname}
 pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
-pkgver=2021.06.30
+pkgver=2022.01.04
```
Comment (MehdiChinoune) [inline mingw-w64-opencl-headers/PKGBUILD:6]:
```suggestion
pkgver=2022.01.04
clhpp_ver=2.0.16
```

File: mingw-w64-opencl-headers/PKGBUILD [hunk 2]
```diff
@@ -12,11 +12,12 @@ mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
...
 license=('Apache 2.0')
 makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
-             "${MINGW_PACKAGE_PREFIX}-ninja")
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cc")
 source=(OpenCL-Headers-${pkgver}.tar.gz::https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v${pkgver}.tar.gz
-        OpenCL-CLHPP-2.0.15.tar.gz::https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2.0.15.tar.gz)
-sha256sums=('6640d590c30d90f89351f5e3043ae6363feeb19ac5e64bc35f8cfa1a6cd5498e'
-            '0175806508abc699586fc9a9387e01eb37bf812ca534e3b493ff3091ec2a9246')
+        OpenCL-CLHPP-2.0.16.tar.gz::https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2.0.16.tar.gz)
```
Comment (MehdiChinoune) [inline mingw-w64-opencl-headers/PKGBUILD:18]:
```suggestion
        OpenCL-CLHPP-${clhpp_ver}.tar.gz::https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v${clhpp_ver}.tar.gz)
```

File: mingw-w64-opencl-headers/PKGBUILD [hunk 3]
```diff
@@ -31,14 +32,14 @@ package() {
 
   DESTDIR=${pkgdir} cmake --install .
 
-  cd "${srcdir}"/OpenCL-CLHPP-2.0.15
+  cd "${srcdir}"/OpenCL-CLHPP-2.0.16
```
Comment (MehdiChinoune) [inline mingw-w64-opencl-headers/PKGBUILD:35]:
```suggestion
  cd "${srcdir}"/OpenCL-CLHPP-${clhpp_ver}
```

File: mingw-w64-opencl-headers/PKGBUILD [hunk 4]
```diff
@@ -31,14 +32,14 @@ package() {
...
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
     ${MINGW_PREFIX}/bin/cmake \
       -G'Ninja' \
       -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
       "${extra_config[@]}" \
       -DBUILD_TESTS=OFF \
-      ../OpenCL-CLHPP-2.0.15
+      ../OpenCL-CLHPP-2.0.16
```
Comment (MehdiChinoune) [inline mingw-w64-opencl-headers/PKGBUILD:42]:
```suggestion
      ../OpenCL-CLHPP-${clhpp_ver}
```

**Pull request merged**